templates/search/index.rs.html

```diff
@@ -7,6 +7,7 @@
   <h1>@i18n!(ctx.1, "Search")</h1>
   <form method="get" id="form">
     <input id="q" name="q" placeholder="@i18n!(ctx.1, "Your query")" type="search">
+      <style type="text/css">-webkit-appearance: none;</style>
```

Comment:
Why don't you add it to the stylesheet directly?

Reply:
Where do you mean, I am not too sure...sorry. In the `.css` files I have adressed every form and `input` field that I felt like it needed to be adressed. I have reformatted the line you higlighted, so it should be all fine now, I guess? :-)

Reply:
(sorry for the late reply)

You actually forgot this one: https://github.com/Plume-org/Plume/pull/560/files#diff-339bac14a232fb01418ef93c2cb5e1ecR6

I think if you add `appearence: none` here, you could remove all the inline styles and have them all in one place.

Reply:
Unfortunately, in some cases, like with imput forms and buttons, there just has to be the -webkit prefix present, otherwise the buttons are disorted when it's just appearence: none, and so the desired effect is not achievable otherwise it seems like...